src/account-link.ts

```diff
@@ -1,6 +1,10 @@
 import prefixForNetwork from './prefix-for-network'
 
-export = function getAccountLink(address: string, network: string): string {
-  const prefix = prefixForNetwork(network)
-  return prefix !== null ? `https://${prefix}etherscan.io/address/${address}` : '';
+export = function getAccountLink(address: string, network: string, customNetworkUrl?: string): string {
```

Comment:
It is a bit unfortunate that `network` is still a required parameter here even when the caller is resolving a custom network.

Maybe resolving a custom network should be a separate function altogether? It doesn't share any common code with resolving a built-in network.

Reply:
So would we call that separate function in a conditional statement or are we exporting that separate function all together?

Reply:
My concern is who will dynamically determine what is a custom url or not. I feel as though we should just take a string and based upon what we have defined we determine what is and isn't a custom url. I could be misinterpreting what you're stating.

Reply:
The caller of this function already has to know whether they're on a custom network or not. They have to know what `network` means, and what `customNetworkUrl` means, and when to pass in each.

I just thought this would be simpler to use of the two cases were separated, such that `getAccountLink(address: string, network: string)` would stay as-is, and we'd have a new function like `customBlockExplorer.getAccountLink(address: string, customNetworkUrl: string)` for the custom network case. Then it's clear for the user of the library what each function does, and there are no extraneous parameters, and the body of the function doesn't end up being two completely distinct branches.